core/sql/exp/exp_math_func.cpp

```diff
@@ -337,8 +338,22 @@ ex_expr::exp_return_type ExFunctionMath::eval(char *op_data[],
 	  **diagsArea << DgString0("LOG");
 	  return ex_expr::EXPR_ERROR;
 	}
-      
-      *(double *)op_data[0] = MathLog(*(double *)op_data[1], err);
+
+      if(3 == getNumOperands())
+    {
```

Comment:
I use space, but somehow the original indentation is not recognized, I see they are the same in my editor, not sure which part of the configuration of vim causes this. Anyway, I tried to re-org it, will push on to see how it looks.